src/components/organisms/AdminVenues/AdminVenues.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from "react";
+import { Button } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+import { AnyVenue, VenueTemplate } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import "./AdminVenues.scss";
+
+interface AdminVenuesProps {
+  venues: WithId<AnyVenue>[];
+}
+
+export const AdminVenues: React.FC<AdminVenuesProps> = ({ venues }) => {
+  const partyVenues = useMemo(() => {
+    return venues?.filter((venue) => venue.template === VenueTemplate.partymap);
+  }, [venues]);
+
+  const hasVenues: boolean = !!partyVenues.length;
+
+  return (
+    <div className="admin-venue">
+      <div className="admin-venue__create-button">
+        <h3>Admin Dashboard</h3>
+        <Button as={Link} to="/admin_v2/venue/creation">
+          Create a new space
+        </Button>
+      </div>
+      {!hasVenues && (
+        <div className="admin-venue__empty-venues">
```

Comment:
Potentially this would still be `admin-venue__cards` but with a `--empty` modifier or similar? As it seems to be the same 'slot' in the UI.

Reply:
This is a separate element on purpose. If venues exist you I render element A, if the venues don't exist I render element B

Reply:
I understand what/how this is being used here, i'm just suggesting that the distinction/separate element for it is probably not needed, as per my above.